apps/testnorge-arena/arena-core/src/main/java/no/nav/registre/arena/core/service/util/ServiceUtils.java

```diff
@@ -130,6 +145,65 @@
         return new ArrayList<>(identerMedAktoerId.keySet()).subList(0, antallNyeIdenter);
     }
 
+    private List<String> filtrerIdenterUtenAktoerIdOgBarnUnder18(
+            Set<String> identer,
+            String miljoe,
+            int antallNyeIdenter,
+            LocalDate tidligsteDato
+    ){
+        var identerUtenArenabruker = filtrerEksisterendeBrukereIArena(identer, miljoe);
+
+        var identerPartisjonert = partisjonerListe(identerUtenArenabruker, PAGE_SIZE);
+
+        List<String> utvalgteIdenter = new ArrayList<>(antallNyeIdenter);
+
+        for (var partisjon : identerPartisjonert) {
```

Comment:
Kompleksiten på denne er for høy. Noe som sonar også reagerer på se. Code smell

Reply:
Ja, så det. Prøver å fikse det nå :)

Reply:
Har fikset :)